google/cloud/forseti/scanner/scanners/enabled_apis_scanner.py

```diff
@@ -135,7 +134,7 @@ def _retrieve(self):
 
         if not enabled_apis_data:
             LOGGER.warn('No Enabled APIs found. Exiting.')
-            sys.exit(1)
+            return iter([])
```

Comment:
This should raise a `NoDataError` exception, 
https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/google/cloud/forseti/common/util/errors.py#L42-L44

Then in the `run()` method, you can do this, which will exit cleanly and early.
```
try:
    enable_apis_data = self._retrieve()
except NoDataError:
    return
all_violations = self._find_violations(enabled_apis_data)
self._output_results(all_violations)
```

Reply:
I've made the modifications to the two scanners.  Please let me know if you have further suggestions.